api_handler.py

```diff
@@ -586,7 +586,48 @@ def get_connector_list_from_db():
                               })
     return result
 
-    
+def parse_web_log(date_str, user_email_dict):
+    cmd = f'sudo cat /usr/lucidum/web/app/logs/logFile.{date_str}.log'
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True)
+    result = []
+    for line_bytes in iter(process.stdout.readline, ''):
+        line = line_bytes.decode()
+        if not line:
+            break
+        if "User successfully logged in" in line:
+            items = line.strip().split(' ')
+            if len(items) > 12:
```

Comment:
Will a system "LOGIN" user have more than 12 characters in the items?

Reply:
oh, this is parsing the log line splitting by space, the 12th item is username and username can't have space